src/services/legal-services.ts

```diff
@@ -89,9 +94,12 @@ export default class LegalServices {
    */
   static async fetchDraftApp (tempRegNumber: string): Promise<any> {
     const url = `businesses/${tempRegNumber}/filings`
-    return axios.get(url)
-      // workaround because data is at "response.data.data"
-      .then(response => response?.data)
+    return (
+      axios
+        .get(url)
+        // workaround because data is at "response.data.data"
+        .then((response) => response?.data)
+    )
```

Comment:
I'm curious why you change the code style one way here and a different way above. Why not leave it as it was?

Reply:
ES lint auto formats the code. I can align the style with what's above.

Reply:
Oh, we've seen this before, but not recently.

Is this what your linting step looks like?

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/97e94f05-f4a9-4da9-8162-6fed4dd6d7c3)

Do you perhaps have a different linter installed globally? Here's what I have:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/ba9248e9-1e06-4dbf-96ad-5d1b2fdbbf45)

Reply:
^^ When I run lint, it doesn't reformat anything.

Reply:
It might be my IDE settings, auto formatting. I have the Prettier extension installed which seems to be doing its thing on save. I will revert the un-related changes.

Reply:
Thanks for reverting. Looks good.